tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html

```diff
@@ -1,38 +1,63 @@
 {% if pages.paginator.num_pages > 1 %}
```

Comment:
Let's try and stick with established naming conventions for things like pagination. The [examples in the Django docs](https://docs.djangoproject.com/en/5.0/topics/pagination/) have used `page_obj` for the current page for years, which tends to work well. 

If you don't want the 'iterable of results' to be called `page_obj` in the main template, then do what Django's generic views do and make the same object available as both `pages` and `page_obj`.